pkg/errors/errorMessage.go

```diff
@@ -155,6 +155,8 @@ var (
 	GALASA_ERROR_BAD_TIME_UNIT_AGE_SPECIFIED    = NewMessageType("GAL1082E: Invalid time unit specified '%s' in the '--age' parameter. Age of the test runs should be specified in the format '{FROM}{TIME-UNIT}:{TO}{TIME-UNIT}' or '{FROM}{TIME-UNIT}', where 'FROM' is a positive, non-zero integer, 'TO' is a non-negative integer, and 'TIME-UNIT' can be 'w' (weeks), 'd' (days) or 'h' (hours). 'FROM' must be greater than 'TO'. 'TO' defaults to 0 if not specified.", 1082, STACK_TRACE_NOT_WANTED)
 	GALASA_ERROR_NO_ARTIFACTS_TO_DOWNLOAD       = NewMessageType("GAL1083E: No artifacts to download for run: '%s'", 1083, STACK_TRACE_NOT_WANTED)
 	GALASA_ERROR_HTTP_RESPONSE_CLOSE_FAILED     = NewMessageType("GAL1084E: Communications failure while closing an HTTP response. '%s'", 1084, STACK_TRACE_WANTED)
+	GALASA_ERROR_INVALID_RESULT_ARGUMENT        = NewMessageType("GAL1085E: Invalid result parameter %s. The possible result values in RAS are: %s", 1085, STACK_TRACE_NOT_WANTED)
+	GALASA_ERROR_QUERY_RESULTNAMES_FAILED       = NewMessageType("GAL1086E: Could not retrieve ruesult names. Reason: '%s'", 1086, STACK_TRACE_WANTED)
```

Comment:
Communications problem between the command-line tool and the target Galasa Ecosystem. The tool could not retrieve the list of valid result names. Reason: '%s'